programs/locker/src/instructions/v2/cancel_vesting_escrow.rs

```diff
@@ -99,20 +97,30 @@ pub fn handle_cancel_vesting_escrow(ctx: Context<CancelVestingEscrow>) -> Result
     drop(escrow);
 
     // Transfer the claimable amount to the recipient
-    transfer_to_user(
+    transfer_to_user_v2(
         &ctx.accounts.escrow,
+        &ctx.accounts.token_mint,
         &ctx.accounts.escrow_token,
         &ctx.accounts.recipient_token,
         &ctx.accounts.token_program,
+        Some(MemoTransferContext {
+            memo_program: &ctx.accounts.memo_program,
+            memo: transfer_memo::TRANSFER_MEMO_CANCEL_VESTING.as_bytes(),
+        }),
         claimable_amount,
     )?;
 
     // Transfer the remaining amount to the sender
```

Comment:
Transfer the remaining amount to the sender -> Transfer the remaining amount to the creator

Reply:
updated, I put those constants inside util::token_2002